guide.md

```diff
@@ -13535,13 +13541,15 @@ See the [Some last OPSEC thoughts][Some last OPSEC thoughts:] section for some t
   [Setup the VPN ProxyVM:]: #setup-the-vpn-proxyvm
   [Setup a safe Browser within Qubes OS (optional but recommended):]: #setup-a-safe-browser-within-qubes-os-optional-but-recommended
   [Setup an Android VM:]: #setup-an-android-vm
-  [3]: #keepassxc-1

```

Comment:
This is why I added precision to the titles: this points to `[3]` which points to `[3]: #keepassxc-1` ... it adds unneeded steps.

Reply:
Still, it reads weird with the redundant titles